README.md

```diff
@@ -7,8 +7,10 @@
 Package   | Description
 ----------|---------------------------------------------------------------------------------------------------------------------
 inclusion | Package inclusion contains functions to generate the blob share commitment from a given blob.
+proto     | Package contains proto definitions and go generated code
```

Comment:
[question] does this proto package actually exist? It looks like the proto definition for BlobTx and IndexWrapper are still inside proto/blob/v1

Reply:
put another way: there is no Go package named `proto`

Reply:
No but here is generated code in the proto directory and that's what I was referring to. I'm happy to remove it

Reply:
I thought this list was just Go packages but I can see an argument for expanding this list to all directories and not just Go packages